### ISSUE TITLE
Display actual rolls in tooltip

### ISSUE THREAD

**Issue by cs96and** [slcao2/FoundryVTT-QuickRolls#36]
Not sure how easy this is to do, but it would be nice if the expanded view (or a cut down version of it) was available as a tooltip when you hover over a roll.  This way you can see the actual dice rolls without having to expand the card.

**Comment by slcao2** [comment]
This one might be a little more difficult since as far as I know, the existing tooltip hover doesn't have the proper stylings to display anything beyond simple text. I can look into it as an enhancement though.

**Comment by cs96and** [comment]
I don't think it needs anything more than simple text.  You could still expand the roll to see the full stylised version
